test/test-applications/integrations/dependency-libs/Samples.NoMultiLoader.Deps/Samples.NoMultiLoader.Deps.csproj

```diff
@@ -1,11 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net45;net461;netstandard2.0;netcoreapp3.1</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard2.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net45;net461;netstandard2.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Any reason we're not including .NET 5.0 here? 

Also, I know it's not from your changes, but why net45 and not net452 like everything else?

Reply:
I think this can use the default frameworks, since it doesn't use framework-specific code

Reply:
Done in bc24eaf9.

Reply:
Actually, these have an empty props so they need to specify target. Fixed in 94de9af8.